src/java.base/share/classes/java/io/BufferedReader.java

```diff
@@ -59,6 +59,11 @@
  * <p> Programs that use DataInputStreams for textual input can be localized by
  * replacing each DataInputStream with an appropriate BufferedReader.
  *
+ * <p> More than one instance of BufferedReader should not be used with the
```

Comment:
Yes, I saw that but did not want to change it in the initial pass(es) as it's too much noise. If things are otherwise quiescent, then I can change it.

Reply:
Fixed in d6831f9 and added a link for `line.separator`.